stripe_official.php

```diff
@@ -1616,7 +1616,6 @@ public function hookHeader()
         $amount = $cart->getOrderTotal();
         $amount = Tools::ps_round($amount, 2);
         $amount = $this->isZeroDecimalCurrency($currency->iso_code) ? $amount : $amount * 100;
```

Comment:
Is it already done line 50 on initContent() method which call this (constructIntentData()) ?